xsd/netex_framework/netex_utility/netex_utility_xml.xsd

```diff
@@ -55,22 +55,21 @@ Rail transport, Roads and Road transport
 		<xsd:documentation>Xml utility types for NeTEX</xsd:documentation>
 	</xsd:annotation>
 	<!-- ======================================================================= -->
-	<xsd:complexType name="MultilingualString">
+	<xsd:complexType name="MultilingualString" mixed="true">
 		<xsd:annotation>
-			<xsd:documentation>Type for a string in a specified language.</xsd:documentation>
+			<xsd:documentation>Either use with simple text (as was used in NeTEx priori to v2.0) or use only Text elements withing the MultilingualString. The old version will be deprecated. Also to take advantage of mixed may not be processable by some data consumers.</xsd:documentation>
 		</xsd:annotation>
+		<xsd:sequence>
+			<xsd:element name="Text" type="TextType" minOccurs="0" maxOccurs="unbounded"/>
+		</xsd:sequence>
+		<xsd:attribute name="lang" type="xsd:string"/>
+		<xsd:attribute name="textIdType" type="xsd:string"/>
+	</xsd:complexType>
+	<xsd:complexType name="TextType">
 		<xsd:simpleContent>
-			<xsd:extension base="xsd:normalizedString">
-				<xsd:attribute name="lang" type="xsd:language">
-					<xsd:annotation>
-						<xsd:documentation>Language of string contents.</xsd:documentation>
-					</xsd:annotation>
-				</xsd:attribute>
-				<xsd:attribute name="textIdType" type="xsd:normalizedString">
-					<xsd:annotation>
-						<xsd:documentation>Resource id of string.</xsd:documentation>
-					</xsd:annotation>
-				</xsd:attribute>
+			<xsd:extension base="xsd:string">
+				<xsd:attribute name="lang" type="xsd:string"/>
```

Comment:
same: I would rather keep the _xsd:language_ type here

Reply:
done